packages/lesswrong/lib/collections/petrovDayActions/schema.ts

```diff
@@ -0,0 +1,28 @@
+
+const schema: SchemaType<"PetrovDayActions"> = {
+  // NOTE: this whole schema is bad, sorry Robert
+  actionType: {
+    type: String,
+    optional: true,
+    nullable: false,
+    canRead: ['guests'],
+    canCreate: ['members'],
+    canUpdate: ['members'],
+  },
+  data: {
+    type: Object,
+    nullable: true,
+    optional: true,
+    canRead: ['guests'],
+    canCreate: ['members'],
+    canUpdate: ['members'],
+  },
+  userId: {
+    type: String,
+    canRead: ['guests'],
+    canCreate: ['members'],
+    canUpdate: ['members'],
```

Comment:
`canUpdate` should probably be `admins` only, `canRead` should probably be `userOwns` and `admins` only.